### PR TITLE
[WIP] update scripts for integration

### PR DESCRIPTION
**Pull request by damiannolan**
Draft PR for router update + additional wallet for hermes relayer.

### CODE REVIEW

**Review by seantking — APPROVED** [review]

File: app/app.go [hunk 1]
```diff
@@ -377,7 +377,7 @@ func New(
 	ibcRouter.AddRoute(icacontrollertypes.SubModuleName, icaControllerIBCModule).
 		AddRoute(icahosttypes.SubModuleName, icaHostIBCModule).
 		AddRoute(ibctransfertypes.ModuleName, transferIBCModule).
-		AddRoute(intertxtypes.ModuleName, interTxIBCModule)
+		AddRoute(intertxtypes.ModuleName, icaControllerIBCModule)
```
Comment (crodriguezvega) [inline app/app.go:380]: Sorry, can you explain me why this change?

Comment (colin-axner) [reply app/app.go:380]: We need to route the packet to the top level of the middleware stack. See [docs](https://github.com/cosmos/ibc-go/blob/main/docs/ibc/middleware/integration.md#integrating-ibc-middleware-into-a-chain)

> An IBC application may be wrapped by many layers of middleware, only the top layer middleware should be hooked to the IBC router, with all underlying middlewares and application getting wrapped by it.
